CONTRIBUTING.md

```diff
@@ -72,3 +72,14 @@ OR
 * The module is under the Node.js foundation Github org
 OR
 * The module is identified as an important module by a Node.js Working Group
+
+#### Procedure
+
+After making sure you adhere to the above requirements, do the
+following:
+
+1. Add the module to
+   [`lib/lookup.json`](https://github.com/nodejs/citgm/blob/master/lib/lookup.json)
+1. Run `npm link`
+1. Make sure the `citgm <module>` tests pass
+1. Commit your changes and open a PR
```

Comment:
@watson I think that can be done in a separate PR, we should get this landed.

Could you add the sentence I suggested anyway (or something similar)?